app/search/forms.py

```diff
@@ -12,20 +15,32 @@
 
 
 class FieldsConstant:
-
     Q = "q"
     SORT = "sort"
     LEVEL = "level"
     GROUP = "group"
     COLLECTION = "collection"
     ONLINE = "online"
     HELD_BY = "held_by"
+    RECORD_DATE_FROM = "rd_from"
+    RECORD_DATE_TO = "rd_to"
+    OPENING_DATE_FROM = "od_from"
+    OPENING_DATE_TO = "od_to"
```

Comment:
These date `rd_from, od_from` feel cryptic. I think we should be aligning these names, closer to the back end API attributes,  `dateCovering`, I'm not aware of API opening date attribute. Do you know what it is?
```suggestion
    COVERING_DATE_FROM = "covering_date_from"
    COVERING_DATE_TO = "covering_date_to"
    OPENING_DATE_FROM = "opening_date_from"
    OPENING_DATE_TO = "opening_date_to"
```